bumpversion/__init__.py

```diff
@@ -615,7 +615,7 @@ def main(original_args=None):
 
         if 'files' in dict(config.items("bumpversion")):
             warnings.warn(
-                "'files =' configuration is will be deprecated, please use [bumpversion:file:...]",
```

Comment:
`PendingDeprecationWarning` means it is not yet deprecated, which means removing 'is' is the appropriate change here.